db/migrate/20181204201916_create_organization_permissions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrganizationPermissions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :organization_permissions do |t|
```

Comment:
default uuid.

I tried looking up a way to automatically add this for us. Do you have a suggestion?